colorizer_data/tests/test_converter.py

```diff
@@ -16,34 +38,23 @@ def feature_array_to_dict(
 
 def validate_data(path: pathlib.Path, data: Union[List[int], List[float]]):
     loaded_data = []
-    with open(path, "r") as f:
-        if path.suffix == ".json":
+    if path.suffix == ".json":
+        with open(path, "r") as f:
             feature_data = json.load(f)
             loaded_data = feature_data["data"]
-        else:
-            feature_data = pd.read_parquet(f)
-            loaded_data = feature_data["data"]
+    else:
+        feature_data = pd.read_parquet(path)
+        loaded_data = feature_data["data"]
 
     assert len(loaded_data) == len(data)
     for i, d in enumerate(data):
         assert abs(loaded_data[i] - d) < 1e-6
 
 
-sample_csv_headers = "ID,Track,Frame,Centroid X,Centroid Y,Continuous Feature,Discrete Feature,Categorical Feature,File Path"
-sample_csv_headers_alternate = "object_id,track,frame,centroid_x,centroid_y,continuous_feature,discrete_feature,categorical_feature,file_path"
-sample_csv_data = """0,1,0,50,50,0.5,0,A,./colorizer_data/tests/assets/test_csv/frame_0.tiff
-    1,1,1,55,60,0.6,1,B,./colorizer_data/tests/assets/test_csv/frame_1.tiff
-    2,2,0,60,70,0.7,2,C,./colorizer_data/tests/assets/test_csv/frame_0.tiff
-    3,2,1,65,75,0.8,3,A,./colorizer_data/tests/assets/test_csv/frame_1.tiff"""
-
-
-def test_handles_simple_csv(tmp_path):
-    csv_content = f"{sample_csv_headers}\n{sample_csv_data}"
-    csv_data = pd.read_csv(StringIO(csv_content))
-    convert_colorizer_data(csv_data, tmp_path / "dataset", use_json=True)
-
-    expected_manifest = tmp_path / "dataset" / "manifest.json"
+def validate_default_dataset(dataset_dir: pathlib.Path, filetype="json"):
```

Comment:
All the validation code previously in `test_handles_simple_csv` was refactored into `validate_default_dataset`.